docassemble/MAVirtualCourt/data/questions/Housing_Temporary_Restraining_Order.yml

```diff
@@ -536,39 +537,42 @@ need:
 decoration: file-download
 question: |
   % if not defined('email_success') or not email_success:
-  Ready to be delivered
+  Review, Download, and Send Form
```

Comment:
I still find this confusing. If we just remove 'and Send Form' I think this would be a lot more clear. The pages each say what you're doing when you interact with them. You can't interact with this page to actually send the form.